CMakeLists.txt

```diff
@@ -2,24 +2,32 @@ cmake_minimum_required(VERSION 3.5.0)
 project(opw_kinematics VERSION 0.1.0 LANGUAGES CXX)
 
 find_package(Eigen3 REQUIRED)
+find_package(cmake_common_scripts REQUIRED)
+
+add_code_coverage_all_targets()
+
 if(NOT EIGEN3_INCLUDE_DIRS)
   set(EIGEN3_INCLUDE_DIRS ${EIGEN3_INCLUDE_DIR})
 endif()
 
-list(FIND CMAKE_CXX_COMPILE_FEATURES cxx_std_11 CXX_FEATURE_FOUND)
+if(OPW_ENABLE_TESTING)
```

Comment:
Maybe a comment of what you are doing here for the mere mortals that will follow you.

Reply:
fixed.